connect_ext_test.go

```diff
@@ -2423,7 +2423,9 @@ func TestClientDisconnect(t *testing.T) {
 			assert.NotNil(t, err)
 			<-gotResponse
 			assert.NotNil(t, handlerReceiveErr)
-			assert.Equal(t, connect.CodeOf(handlerReceiveErr), connect.CodeCanceled)
+			if !assert.Equal(t, connect.CodeOf(handlerReceiveErr), connect.CodeCanceled) {
+				t.Logf("handlerReceiveErr: %v", handlerReceiveErr)
+			}
```

Comment:
`assert.Sprintf` is for this:

```suggestion
			assert.Equal(
				t,
				connect.CodeOf(handlerReceiveErr),
				connect.CodeCanceled,
				assert.Sprintf("handerReceiveErr: %v", handlerReceiveErr),
			)
```

At some point, we may want to migrate assertions to `testify` or just use `cmp` directly. I like `assert`-style utility libraries, but it's not worth the mental overhead of having a special approach just for this one repository.